hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -3518,21 +3518,21 @@ hazelcast:
   #
```

Comment:
`<jet>` phrase is remaining above at L:3512. We can adapt it to the general style of this file and replace it with `"jet"`.

Reply:
Added. Converted a couple of other places into quotes as well. There's more in the YAML file, but I didn't feel like going over all of it.